### PR TITLE
Release crypto-nodejs beta1

### PR DESCRIPTION
**Pull request by gnunicorn**

### CODE REVIEW

**Comment by gnunicorn** [comment]
It's done https://www.npmjs.com/package/@matrix-org/matrix-sdk-crypto-nodejs

**Comment by Hywan** [comment]
I bet it's safe to assume we can close it?

**Review by Hywan — APPROVED** [review]

File: bindings/matrix-sdk-crypto-nodejs/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@matrix-org/matrix-sdk-crypto-nodejs",
-    "version": "0.1.0",
+    "version": "0.1.0-beta.0",
```
[the remark this answers is not in the document]
Comment (gnunicorn) [reply bindings/matrix-sdk-crypto-nodejs/package.json:3]: wait, yeah, it should already have been updated. odd. I have to check it.

File: .github/workflows/prep-crypto-nodejs-release.yml [hunk 2]
```diff
@@ -91,18 +93,25 @@ jobs:
...
         with:
-          branch: "gh-action/release-${{ env.TAG_PREFIX }}${{ inputs.version }}"
           title: "Preparing Release ${{ env.TAG_PREFIX }}${{ inputs.version }}"
           body: |
-            Automatic Pull-Request to merge release ${{ env.TAG_PREFIX }}${{ inputs.version }} back into main
+            Automatic Pull-Request to merge release ${{ env.TAG_PREFIX }}${{ inputs.version }}
+
+  trigger-release:
+    # and trigger the tagging release workflow
+    uses: matrix-org/matrix-rust-sdk/.github/workflows/release-crypto-nodejs.yml@ben-release-crypto-nodes-beta0
```
[the remark this answers is not in the document]
Comment (gnunicorn) [reply .github/workflows/prep-crypto-nodejs-release.yml:113]:
```suggestion
    uses: matrix-org/matrix-rust-sdk/.github/workflows/release-crypto-nodejs.yml@main
```

**Pull request merged**